package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "power-lifting",
+  "version": "1.0.0",
+  "description": "Pump weights. Win prizes. Defeat your rivals. All for the sake of...",
+  "source": "index.html",
+  "staticFiles": {
+    "staticPath": "assets",
```

Comment:
`npm run dev` is failing because it says this path does not exist

Reply:
Fixed?